src/queries/recently-merged-prs-project.ts

```diff
@@ -0,0 +1,48 @@
+import { gql } from "apollo-boost";
+import { client } from "../graphql-client";
+import { GetRecentlyMergedPRsProject } from "./schema/GetRecentlyMergedPRsProject";
+
+const GetRecentlyMergedPRsProjectQuery = gql`
+  query GetRecentlyMergedPRsProject {
+    repository(owner: "DefinitelyTyped", name: "DefinitelyTyped") {
+      project(number: 5) {
+        columns(first: 5) {
+          nodes {
+            name
+            cards(last: 100) {
+              nodes {
+                id
+                updatedAt
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+`;
```

Comment:
There isn't a way via the GraphQL API to just get the cards for a specific column, so this is more wasteful than I'd like but it gets the job done.